src/common/transformations/tests/common_optimizations/transpose_sinking_test.cpp

```diff
@@ -195,7 +195,21 @@ class TransposeSinking : public ov::test::TestsCommon,
 
 private:
     std::shared_ptr<Node> get_reduction(NodeTypeInfo reduction_type_info, const OutputVector& inputs, bool keep_dims) {
-        auto reduction = ngraph::helpers::getNodeSharedPtr(reduction_type_info, inputs);
+        std::shared_ptr<Node> reduction;
+        for (const auto& it : get_available_opsets()) {
+            const auto& opset = it.second();
+            if (opset.contains_type(reduction_type_info)) {
+                reduction = std::shared_ptr<Node>(opset.create(reduction_type_info.name));
```

Comment:
In theory the `reduction` pointer can be overwritten for if different `opset` if contains same type, is it OK?
If this is not possible maybe after first match the scanning of `opset` should stop and exit from the loop?

Reply:
The purpose of the change is to remove extra functions from `ngraph::helpers`, so I just unwrapped implementation as is here

Your thoughts are right, but I don't want to change implementation because it's for tests only and it works